docs/GE_Checklist.md

```diff
@@ -6,40 +6,138 @@ to be taken in order to publish a new FIWARE GE.
     [Working with the FIWARE catalogue](https://wiki.fiware.org/Working_with_the_FIWARE_catalogue)
 -   Mirroring GitHub Webhook properly configured. See
     [webhooks](repo_webhook.md) (Ask FF to configure the mirroring).
--   **ReadtheDocs** portal available and styled using the FIWARE CSS. See
+-   **ReadtheDocs** project available and styled using the FIWARE CSS and
+    Chapter CSS. See
     [fiware-orion.readthedocs.org](https://fiware-orion.readthedocs.org) as an
     example.
--   See
+-   Automatic documentation generation **MUST** be configured. See
     [mkdocs.yml](https://github.com/telefonicaid/fiware-orion/blob/master/mkdocs.yml)
-    as a reference on how to configure **ReadtheDocs** for automatic
-    documentation generation.
--   Analytics should be configured for the **ReadtheDocs** portal as described
+    as a reference on how to configure **ReadtheDocs**
+-   Analytics **MUST** be configured for the **ReadtheDocs** portal as described
     by [analytics](analytics_readthedocs.md)
--   Docker image is available in
+-   Docker image **MUST** be available in
     [FIWARE's Dockerhub account](https://hub.docker.com/Dockerhub)
--   FIWARE Development (Must) Guidelines are met. See
-    [Development Guidelines](development.md)
--   A Stack Overflow tag for the component is registered (Ask FF Staff)
--   Where it is suitable to provide a global instance of a service, a public
-    instance is deployed in the FIWARE Lab and maintained by the FF team (Ask FF
-    Staff)
--   Roadmap is available at one of the following:
+-   Roadmap **MUST** be available at one of the following:
     -   Promoted on GitHub and linked in the
         [Current Roadmap](https://wiki.fiware.org/Current_Supported_Features_and_Roadmap_in_FIWARE)
     -   Promoted in a publicly available document on the web and linked at
         [Current Roadmap](https://wiki.fiware.org/Current_Supported_Features_and_Roadmap_in_FIWARE)
     -   Full text found within the
         [Current Roadmap](https://wiki.fiware.org/Current_Supported_Features_and_Roadmap_in_FIWARE)
--   Release schedule is aligned with
+-   Release schedule **MUST** be aligned with
     [release requirements](GE_Requirements.md#releases)
--   A tutorial is provided (See some examples at
-    [fiware-tutorials.readthedocs.io](http://fiware-tutorials.readthedocs.io/en/latest))
--   Provide access to the corresponding information in order to see your
-    component reflected in the [ monitoring
+-   Access **MUST** be provided to the corresponding information in order to see
+    your component reflected in the [ monitoring
     tool](https://docs.google.com/spreadsheets/d/1yyZNUlAPDcqjnD-gIoGOd5SZfVDJXO36G75xTDL0HgA/edit#gid=0
     FIWARE - TSC Enablers Dashboard) (Ask FF Staff)
+
+It is accepted that Incubated Enablers may not currently comply with all
+"**MUST**" requirements of the [Development Guidelines](development.md), but are
+expected to be working towards fulfilling them.
```

Comment:
in what timeframe?

Reply:
NTC - Left vague as it will depend on the enabler. Some may need more time than others.